community-operators/openshift-pipelines-operator/0.10.6/openshift-pipelines-operator.v0.10.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,400 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "operator.tekton.dev/v1alpha1",
+          "kind": "Config",
+          "metadata": {
+            "name": "name.must.be-cluster"
+          },
+          "spec": {
+            "targetNamespace": "openshift-pipelines"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Developer Tools, Integration & Delivery
+    certified: "false"
+    containerImage: quay.io/openshift-pipeline/openshift-pipelines-operator:v0.10.5
```

Comment:
Shouldn't containerImage reference tag v0.10.6 not v0.10.5?

Reply:
@ron1 Thanks, i have fixed it.

Looks like it is a bug in the `operator-sdk genereate csv --from-version --csv-version` command.
I had to fix it manually before as well. I will confirm it and file a bug.